src/mongo/installer/compass/Install-Compass.ps1.in

```diff
@@ -33,6 +33,11 @@ Remove-Item $CompassExe -ErrorAction:Ignore
 
 try {
     Write-Output "Downloading Compass from $CompassUrl"
+    
+    # Default PowerShell SecurityProtocol does not support Tls1.2 (required by domain)
+    if ([Net.ServicePointManager]::SecurityProtocol.ToString() -NotMatch "Tls12") {
+        [Net.ServicePointManager]::SecurityProtocol += [Net.SecurityProtocolType]::Tls12 

```

Comment:
You need to just set the SecurityProtocol to Tls12, you can't "add it" it throws the following error:

```
Method invocation failed because [System.Net.SecurityProtocolType] doesn't contain a method named 'op_Addition'.
At Z:\mongo\src\mongo\installer\compass\Install-Compass.ps1:36 char:9
+         [Net.ServicePointManager]::SecurityProtocol += [Net.SecurityProtocolType ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (op_Addition:String) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : MethodNotFound
```

Reply:
Additionally your commit message needs to be updated to include the Jira ticket so:

SERVER-35936 Allow PowerShell to connect using TLS 1.2

Reply:
I've made the change requested. 

I also apologize for doing this work on master, as I was unfamiliar with the work process for mongo. Would you prefer me to resubmit under a branch with the usual naming convention for Mongo PRs or is this fine as is?

Reply:
This is fine as is, most of us work off master anyway. 

I'm going to do some manual testing on this fix but I don't see any reason it wouldn't work. Thanks for your help we appreciate you taking care of this for us.